### PR TITLE
Roadmap issue template and README section about releases

### PR DESCRIPTION
**Pull request by bauersimon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,10 +213,17 @@ To add new tasks to the benchmark, add features, or fix bugs, you'll need a deve
...
--   Add more features and fix bugs in the evaluation, development environment, or CI: [best to have a look at the list of issues](https://github.com/symflower/eval-dev-quality/issues).
+- Add more files to existing language repositories.
+- Add more repositories to languages.
+- Implement another language and add repositories for it.
+- Implement new tasks for existing languages and repositories.
+- Add more features and fix bugs in the evaluation, development environment, or CI: [best to have a look at the list of issues](https://github.com/symflower/eval-dev-quality/issues).
 
 If you want to contribute but are unsure how: [create a discussion](https://github.com/symflower/eval-dev-quality/discussions) or write us directly at [markus.zimmermann@symflower.com](mailto:markus.zimmermann@symflower.com).
+
+# How to do a release?
```
Comment (zimmski) [inline README.md:224]: Document: master should be always in a state that we can do a release.

File: .github/ISSUE_TEMPLATE/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+name: Roadmap issue
+about: Use this template for tracking release roadmaps.
+title: "Roadmap for vXXXXX"
+labels: roadmap
+assignees: zimmski
+---
+
```
Comment (zimmski) [inline .github/ISSUE_TEMPLATE/roadmap.md:8]: Do a goal-summary on what we want to do for the release: reasons for more available scores.

Comment (zimmski) [reply .github/ISSUE_TEMPLATE/roadmap.md:8]: - Goals are based on "main topics" (at least one) towards extending the benchmark on a score-base
- A new tool, a new model, ... is not release-worthy, but it is publish-worthy i.e. how is that new model doing with the benchmark?

Comment (zimmski) [reply .github/ISSUE_TEMPLATE/roadmap.md:8]: Content for a deep-dive blog post:
- Content is about giving others an insight of our experiences: everyone who reads, should learn something
- Findings / Problems. if you are surprised, it is the best content!
  - Journal about what went well
  - Journal about what went not well
- For every finding immediately write down a few bullet points why it is interesting, give examples(!)

**Comment by bauersimon** [comment]
Should we also add our habits of testing new features manually and on our server instance? And with like... Openrouter, Ollama and i.e. fireworks? @zimmski

**Pull request merged**